kubelet/datadog_checks/kubelet/cadvisor.py

```diff
@@ -182,6 +184,8 @@ def _update_container_metrics(self, instance, subcontainer, pod_list, pod_list_u
             if pod_list_utils.is_excluded(cid):
                 self.log.debug("Filtering out " + cid)
                 return
+            if '://' in cid:
+                cid = '://'.join(['container_id', cid.split('://')[1]])
```

Comment:
Please make this a utility function in https://github.com/DataDog/integrations-core/blob/master/kubelet/datadog_checks/kubelet/common.py